templates/projects/js/annotation_guidelines_button.js

```diff
@@ -1,7 +1,20 @@
 <script>
-$(".collapse-guidelines-button").on('click', function(){
-    $(this).text(function(i, v){
-        return (v.startsWith('Show') ? 'Hide' : 'Show') + ' Instructions';
-    });
+const guidelines = $("#collapse-guidelines")
+const button = $(".collapse-guidelines-button")
+const key = '{{project.short_name}}' + "-guidelines-state"
+
+if (localStorage.getItem(key) === "closed") {
+  guidelines.collapse("hide")
+  button.text('Show Instructions')
+}
```

Comment:
Looks like if block guarantees that the guidelines are hidden when it is set in local storage for a given project. Any reason there's no else block that ensures guidelines visible in other case(else part)?

Reply:
The guidelines are visible by default in the html, so no need to set them with js. See [here](https://github.com/bloomberg/pybossa-default-theme/blob/main/templates/annotation_guidelines_button.html).
The default button text is `Hide Instructions` and guidelines div has the property `aria-expanded="true"`

Reply:
Thanks for confirming!